content/millennium/r4/clinical/summary/family-member-history.md

```diff
@@ -15,10 +15,74 @@ The FamilyMemberHistory resource represents family history for a given patient.
 
 The resource should be leveraged as a snapshot in time and new data should be consistently retrieved through the API rather than stored within an application. There may be instances where a relative had an associated condition removed from the patient’s chart which would then no longer be exposed on a subsequent transaction. It is crucial that the data be refreshed with every interaction.
 
+It is recommended that an update to FamilyMemberHistory contain all relevant data that previously existed on the resource to ensure that data remains populated in the patient's chart (e.g., FamilyMemberHistory.sex, FamilyMemberHistory.bornDate, FamilyMemberHistory.condition.onsetDate, etc.). The absence of this optional information may result in the removal of data from the patient's chart.
+
```

Comment:
the jira also says:
Implementation Details - Need to add something along the lines of:

During a client implementation, confirm what vocabulary is supported. Best practice is to leverage a single vocabulary and the Model recommendation is for clients to leverage SNOMED-CT (https://wiki.cerner.com/display/public/reference/Configure+Family+History)

Not sure if we need to say that here?